source/help/getting-started/managing-members.rst

```diff
@@ -1,77 +1,97 @@
-# Managing Members  
-___
+.. _managing-members:
 
-## Inviting Team Members
+Managing Members
+================
+
+Inviting Team Members
+---------------------
 
 If user creation is enabled by the System Admin, there are two methods to invite new members after team creation: sending direct invites and sharing the team invite link.
 
 If a user already has an account on the site but they are not on your team, you can also send them an invitation. If they are signed in, they will be automatically added to your team when they click the invitation link.
 
-#### Direct Invites  
+Direct Invites
+~~~~~~~~~~~~~~
 
 Direct invites are invitation emails sent from your teams server directly to the invited members email address. A link within the invitation email directs the users to an account creation page.
 
-From the three dot **Main Menu** in the Mattermost header, select **Invite New Member**. Enter a new members email address along with their first and last name. Click **Add another** to send more direct invites from the same dialog.
+From the three dot **Main Menu** in the Mattermost header, select **Invite New Member**. Enter a new member's email address along with their first and last name. Click **Add another** to send more direct invites from the same dialog.
 
-#### Team Invite Link  
+Team Invite Link
+~~~~~~~~~~~~~~~~
 
-The team invite link is a unique URL that directs users to an account creation page for your team. Your team invite link can be used by any number of users and does not change unless it is re-generated by a Team or System Admin via **Team Settings** > **General** > **Invite Code**. For example, the team invite link can be included in a company wide email to invite all employees to join a Mattermost team.
+The team invite link is a unique URL that directs users to an account creation page for your team. Your team invite link can be used by any number of users and does not change unless it is re-generated by a Team or System Admin via **Team Settings > General > Invite Code**. For example, the team invite link can be included in a company wide email to invite all employees to join a Mattermost team.
```

Comment:
"... page for your team." > "... page for a team."

"Your team invite link..." > "A team invite link..."